dcap_quoteverify/types/README.md

```diff
@@ -0,0 +1,24 @@
+# MobileCoin's Rust wrappers for the Intel SGX SDK
+
+[![mc-sgx-dcap-quoteverify-sys-types][crate-image]][crate-link]
+![License][license-image]
+[![Project Chat][chat-image]][chat-link]
+
+[![Docs Status][docs-image]][docs-link]
+[![CodeCov Status][codecov-image]][codecov-link]
+[![dependency status][deps-image]][deps-link]
+
+This crate contains the rust type wrappers for the 
+`mc-sgx-dcap-quotevery-sys-types` crate.
```

Comment:
It's kind of an intermediate crate that others could use to write a remote attestation library, so it should probably include more detailed explanation of what it is, links to the Intel descriptions of how DCAP works and what this lib is for, etc.

It doesn't need to be super involved, but here's some examples I like:
- [`blake2`](https://github.com/RustCrypto/hashes/blob/master/blake2/README.md) -- it's a similar sort of low-level  library, and it tells you how it handles MSRV and SemVer.
- [`serde`](https://github.com/serde-rs/serde/blob/master/README.md) -- I like the "and here's links to actual information" section at the top.

Reply:
add more info to the readme, it still feels a bit light, but this is just the types...